src/pages/team.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC } from 'react'
+import SimpleLayout from '@layouts/Simple'
+import { graphql } from 'gatsby'
+import DomainDesktop from '@components/team/DomainDesktop'
+import DomainMobile from '@components/team/DomainMobile'
+
+type Props = {
+  data: {
+    allDaTeamMember: {
+      nodes: any[]
```

Comment:
suggestion(blocking): specify types based on graphql query.  see `/src/page/index.tsx` and `/src/page/regions/index.tsx` for examples